en/presenters.texy

```diff
@@ -685,9 +685,100 @@ You don't have to mark subcomponents as persistent, they are persistent automati
+
+.[note]
+If an error has occurred while the error presenter is being executed, Tracy will show only a simple error page.
+
+catchExceptions
+===============
+
+Default value: `TRUE` in production mode, `FALSE` in debug mode
+
+This parameter decides if the error presenter will be invoked. By changing this value to `TRUE` in the debug mode we can test functionality of the error presenter.
+
+mapping
+=======
+
+Default value: <code>\*: \*Module\\*Presenter</code> (in the sandbox <code>\*: App\\*Module\Presenters\\*Presenter</code>)
+
+This parameter contains an array with rules for converting presenter name to the class name. In the key we define a module name or `*` as a fallback for other modules. The value contains the rule mask:
+/--code neon
+application:
+	mapping:
+		Admin: App\Admin\*Module\*Presenter
+		*: App\*Module\Presenters\*Presenter
+\--
+
+
+The presenter `Front:Blog:Article` is contained inside the Front module. Since this module has no explicit mapping, the mapping with asterisk will be used and the class will be called `App\FrontModule\BlogModule\Presenters\ArticlePresenter`. The presenter `Admin:User:Edit` is part of the `Admin` module which has separate mapping so the name of the presenter class will be `App\Admin\UserModule\EditPresenter` (note that since the `Admin` is already in the mapping definition, it will be stripped from the resulting class name).
+
+silentLinks
+===========
+
+Default value: `FALSE`
+
+This option specifies Nette behaviour in the debug mode when a link generation fails. With `FALSE` value Nette will trigger `E_USER_WARNING`. Changing this configuration option to `TRUE` will suppress this error. In production environment, E_USER_WARNING will be always triggered.
+
+We can also adjust this behaviour using class property [Presenter::$invalidLinkMode|#Invalid links].
+
+scan*
+=====
+
+Nette allows automatic registration of presenters as services. This step improves presenter creation speed. There are few configuration options that adjust where and how Nette looks for them.
+
+scanDirs
+--------
+
+Default value: `%appDir%` (`/app`)
+
+This value contains an array with directories where Nette looks for the presenters. If we specify another value in the configuration, it will be merged with the default one:
+
+/--code neon
+application:
+	scanDirs:
+		- foo
+\--
+
+In this case `scanDirs` will contain values `%appDir%` and `foo`. If we want to disable or overwrite this configuration option, we need to use `!` modifier:
+/--code neon
+application:
+	scanDirs!:
+		- foo
+\--
+
+Now `scanDirs` will contain only `foo`.
+
+scanComposer
+------------
+
+Default value: `TRUE`
+
+This parameter specifies if Nette should look for presenters in the Composer classmap.
+
+scanFilter
+----------
+
+Default value: `Presenter`
+
+Specifies the string, which must be included in the file and class name.
```

Comment:
Dle kodu to opravdu vypada, ze "Presenter" musi byt i v nazvu souboru: https://github.com/nette/application/blob/master/src/Bridges/ApplicationDI/ApplicationExtension.php#L130